rb/lib/selenium/webdriver/common/service.rb

```diff
@@ -87,6 +86,20 @@ def uri
         @uri ||= URI.parse("http://#{@host}:#{@port}")
       end
 
+      protected
+
+      def start_process
+        server_command = [@executable_path, "--port=#{@port}", *@extra_args]
+        @process       = ChildProcess.build(*server_command.compact)
+
+        if $DEBUG
+          @process.io.inherit!
+          puts "Process Starting: #{server_command}"
+        end
+        @process.leader = true
```

Comment:
It applied to all but 2 drivers, so I made it default and overrode it in the subclass for the two applicable `Service` classes. Just trying to reduce duplicated code. I might have gone too far, so I'm ok to put it back if you think it would be more clear.